lib/lotus/helpers/html_helper/self_closing_html_node.rb

```diff
@@ -0,0 +1,25 @@
+module Lotus
+  module Helpers
+    module HtmlHelper
+      class SelfClosingHtmlNode
+        def initialize(name, attributes)
+          @name       = name
+          @attributes = attributes
+        end
+
+        def to_s
+          %(<#{ @name }#{attributes}>)
```

Comment:
I could see that there is no closing `/` for this tag. So does this mean we only support HTML5? What about XHTML?